README.md

```diff
@@ -100,7 +121,25 @@ Each lion should have...
 Create a new lion instance with the name "Simba".
 
 ```ruby
-# code here
+class Animal
+  attr_reader :name, :greet
+  def initialize (name, greet)
+    @name = name
+    @greet = greet
+      puts "Pumba"
+  end
+end
+
+class Lion < Animal
+  attr_reader :name, :greet
+  def initialize (name, greet)
+    @name = name
+    @greet = greet
+  if name == Simba
+    puts  Lion && King
+  end
+  end
+end
 ```
 
```

Comment:
Here's the solution in full

```rb
class Animal
  attr_accessor :name

  def initialize name
    @name = name
  end

  def greet
    puts "Hi, my name is #{@name}."
  end
end

class Lion < Animal
  attr_accessor :king
  @@pack = []

  def initialize name
    super(name)
    if name == "Simba"
      @king = true
    else
      @king = false
    end
    @@pack << self
  end
end
```
Set up a one-on-one with me, if you want to walk through how this is done.